.github/configs/kubeconform.yml

```diff
@@ -0,0 +1,21 @@
+# Kubeconform configuration file
```

Comment:
This file has been just only moved from the `.github/kubeconform.yml` to `.github/configs/kubeconform.yml`, seems due to rebasing, GitHub is unable to see that this was a refactor instead of an addition.